composer.json

```diff
@@ -79,6 +79,7 @@
         "drupal/menu_link_weight": "^1.0-beta3",
         "drupal/menu_position": "~1.0@alpha",
         "drupal/metatag": "~1.3",
+        "drupal/paragraphs": "dev-1.x#9258f0985211427445604ae932df240d363d414d",
```

Comment:
Can bump when https://www.drupal.org/project/paragraphs/issues/2832021 is in a stable.